daemon/execdriver/native/driver.go

```diff
@@ -51,6 +52,12 @@ func NewDriver(root, initPath string, options []string) (*driver, error) {
 		return nil, err
 	}
 
+	if apparmor.IsEnabled() {
+		if err := hasAppArmorProfileLoaded("docker-default"); err != nil {
+			return nil, fmt.Errorf("AppArmor enabled on system, but docker-default profile not loaded. Install policies from contrib/apparmor (https://github.com/docker/docker/tree/master/contrib/apparmor)")
```

Comment:
---> This might be shorter --- I don't think you lose anything by using the shortened URL

AppArmor is enabled but the docker-default profile is not loaded. Install policies from contrib/apparmor (https://goo.gl/CjGtgC) and try again."